cli/compose/loader/loader_multiple_test.go

```diff
@@ -0,0 +1,315 @@
+					"options": map[string]interface{}{
+						"timeout":      "360",
+						"pretty-print": "on",
+					},
+				},
+			},
+			expected: &types.LoggingConfig{
+				Driver: "json-file",
+				Options: map[string]string{
+					"frequency":    "2000",
+					"timeout":      "360",
+					"pretty-print": "on",
+				},
+			},
+		},
+		{
+			name: "no_driver",
+			loggingBase: map[string]interface{}{
+				"logging": map[string]interface{}{
+					"options": map[string]interface{}{
+						"frequency": "2000",
+						"timeout":   "23",
+					},
+				},
+			},
+			loggingOverride: map[string]interface{}{
+				"logging": map[string]interface{}{
+					"options": map[string]interface{}{
+						"timeout":      "360",
+						"pretty-print": "on",
+					},
+				},
+			},
+			expected: &types.LoggingConfig{
+				Options: map[string]string{
+					"frequency":    "2000",
+					"timeout":      "360",
+					"pretty-print": "on",
+				},
+			},
+		},
+		{
+			name: "no_override_options",
+			loggingBase: map[string]interface{}{
+				"logging": map[string]interface{}{
+					"driver": "json-file",
+					"options": map[string]interface{}{
+						"frequency": "2000",
+						"timeout":   "23",
+					},
+				},
+			},
+			loggingOverride: map[string]interface{}{
+				"logging": map[string]interface{}{
+					"driver": "syslog",
+				},
+			},
+			expected: &types.LoggingConfig{
+				Driver: "syslog",
+			},
+		},
+		{
+			name:        "no_base",
+			loggingBase: map[string]interface{}{},
+			loggingOverride: map[string]interface{}{
+				"logging": map[string]interface{}{
+					"driver": "json-file",
+					"options": map[string]interface{}{
+						"frequency": "2000",
+					},
+				},
+			},
+			expected: &types.LoggingConfig{
+				Driver: "json-file",
+				Options: map[string]string{
+					"frequency": "2000",
+				},
+			},
+		},
+	}
+
+	for _, tc := range loggingCases {
+		t.Run(tc.name, func(t *testing.T) {
+			configDetails := types.ConfigDetails{
+				ConfigFiles: []types.ConfigFile{
+					{
+						Filename: "base.yml",
+						Config: map[string]interface{}{
+							"version": "3.4",
+							"services": map[string]interface{}{
+								"foo": tc.loggingBase,
+							},
+						},
+					},
+					{
+						Filename: "override.yml",
+						Config: map[string]interface{}{
+							"version": "3.4",
+							"services": map[string]interface{}{
+								"foo": tc.loggingOverride,
+							},
+						},
+					},
+				},
+			}
+			config, err := Load(configDetails)
+			require.NoError(t, err)
+			require.Equal(t, &types.Config{
+				Filename: "base.yml",
+				Services: []types.ServiceConfig{
+					{
+						Name:        "foo",
+						Logging:     tc.expected,
+						Environment: types.MappingWithEquals{},
+					},
+				},
+				Networks: map[string]types.NetworkConfig{},
+				Volumes:  map[string]types.VolumeConfig{},
+				Secrets:  map[string]types.SecretConfig{},
+				Configs:  map[string]types.ConfigObjConfig{},
+			}, config)
+		})
+	}
+}
+
+func TestSimpleLoad(t *testing.T) {
```

Comment:
`TestLoadMultipleConfigs`  so it's clear how this is different from `TestLoad`, and `TestFullExample` (which needs to be renamed to `TestLoadFull`).